WinImageBuilder.psm1

```diff
@@ -582,15 +574,6 @@ function Create-VirtualSwitch
     return $sw
 }
 
-function Check-Prerequisites

```

Comment:
Can you be more specific about "Removed the Get-WindowsOptionalFeature check because it failed in some conditions". What conditions? I don't think this check should be removed just because it fails in some conditions, rather fix the check.